vendor/github.com/openshift/library-go/pkg/operator/csi/credentialsrequestcontroller/credentials_request_controller.go

```diff
@@ -89,7 +90,7 @@ func (c CredentialsRequestController) sync(ctx context.Context, syncContext fact
 	}
 
 	// if clusterCloudCredential is in manual mode, do not sync cloud credentials
-	if clusterCloudCredential.Spec.CredentialsMode == opv1.CloudCredentialsModeManual {
+	if clusterCloudCredential.Spec.CredentialsMode == opv1.CloudCredentialsModeManual && os.Getenv("ROLEARN") == "" {
```

Comment:
What if the env is set and credentials mode is not manual? Is that a scenario possible? If so, we might need do `s/&&/||/g`